pageserver/src/tenant/timeline/eviction_task.rs

```diff
@@ -54,9 +66,12 @@ impl Timeline {
             }
         }
 
+        // FIXME: is this really unexpected downloadbehaviour in the presence of disk based
+        // eviction?
```

Comment:
disk based eviction is supposed to be emergency-only, so, I think we can
1. remove this fixme and
2. add a remark in line 71 below